README.md

```diff
@@ -2,126 +2,55 @@
 
 ![Build and Test](https://github.com/Jigsaw-Code/outline-server/actions/workflows/build_and_test_debug.yml/badge.svg?branch=master) [![Mattermost](https://badgen.net/badge/Mattermost/Outline%20Community/blue)](https://community.internetfreedomfestival.org/community/channels/outline-community) [![Reddit](https://badgen.net/badge/Reddit/r%2Foutlinevpn/orange)](https://www.reddit.com/r/outlinevpn/)
 
-This repository has all the code needed to create and manage Outline servers on
-DigitalOcean. An Outline server runs instances of Shadowsocks proxies and
-provides an API used by the Outline Manager application.
+Outline Server is a tool to create and manage your own VPN servers using Shadowsocks proxies. If you prefer a graphical interface, you can use the [Outline Manager](https://github.com/Jigsaw-Code/outline-apps/).
 
-Go to https://getoutline.org for ready-to-use versions of the software. **To join our Outline Community, [sign up for the IFF Mattermost](https://internetfreedomfestival.org/wiki/index.php/IFF_Mattermost).**
+**Components:**
 
-## Components
+- **Outline Server** ([`src/shadowbox`](src/shadowbox)): The core proxy server that runs and manages [outline-ss-server](https://github.com/Jigsaw-Code/outline-ss-server/), a Shadowsocks backend. It provides a REST API for access key management.
 
-The system comprises the following components:
+- **Metrics Server** ([`src/metrics_server`](src/metrics_server)): A REST service for optional, anonymous metrics sharing.
 
-- **Outline Server**: a proxy server that runs a Shadowsocks instance for each
-  access key and a REST API to manage the access keys. The Outline Server runs
-  in a Docker container in the host machine.
+**Join the Outline Community** by signing up for the [IFF Mattermost](https://wiki.digitalrights.community/index.php?title=IFF_Mattermost)!
 
-  See [`src/shadowbox`](src/shadowbox)
+## Shadowsocks and Anti-Censorship
 
-- **Outline Manager:** an [Electron](https://electronjs.org/) application that
-  can create Outline Servers on the cloud and talks to their access key
-  management API to manage who has access to the server.
+Outline's use of Shadowsocks means it benefits from ongoing improvements that strengthen its resistance against detection and blocking.
 
-  See [`src/server_manager`](src/server_manager)
+**Key Protections:**
 
-- **Metrics Server:** a REST service that the Outline Server talks to
-  if the user opts-in to anonymous metrics sharing.
+- **AEAD ciphers** are mandatory.
+- **Probing resistance** mitigates detection techniques.
+- **Protection against replayed data.**
+- **Variable packet sizes** to hinder identification.
 
-  See [`src/metrics_server`](src/metrics_server)
+[Read more](docs/shadowsocks.md) about Shadowsocks resistance against detection and blocking.
```

Comment:
```suggestion
See [Shadowsocks resistance against detection and blocking](docs/shadowsocks.md).
```

You can name the content in the link instead, which is usually clearer and preferred by search.

Reply:
Done.